xxx-uv_link_t.md

```diff
@@ -0,0 +1,67 @@
+| Title  | Port core to uv_link_t      |
+|--------|-----------------------------|
+| Author | @indutny                    |
+| Status | DRAFT                       |
+| Date   | 2016-06-03                  |
+
+## 1. Proposal
+
+I propose to replace `StreamBase` with [`uv_link_t`][0], and port existing
+classes that inherit from `StreamBase`.
+
+## 2. Current state of C++ Streams in Node
+
+The fast HTTP and TLS protocol implementation in Node core depends on so called
+`StreamBase` C++ class. This class is an analog of JavaScript streams in C++.
+The main ideas behind `StreamBase` are to:
```

Comment:
Does this mean you'll be removing associated classes like `StreamReq` and `StreamResource`?